nixos/default.nix

```diff
@@ -285,6 +295,12 @@ in {
         type = types.bool;
         default = false;
       };
+
+      url = mkOption {
```

Comment:
Hm, I'd prefer to split this up into a two-field setting, `protocol`, `host`. Then we can construct the default URL and don't rely on the semantics of a required partial URL specification here.

Also, I think it'd make sense to put these settings under a config key so they don't collide with downstream settings; call it `urlParts.protocol` and `urlParts.host`?